src/wallet/wallet2.h

```diff
@@ -494,6 +494,7 @@ namespace tools
       bool double_spend_seen;                                    // True if the key image(s) for the transfer have been seen before.
       uint64_t confirmations;                                    // Number of block mined since the block containing this transaction (or block height at which the transaction should be added to a block if not yet confirmed).
       uint64_t suggested_confirmations_threshold;
+      bool checkpointed = false; // If the transfer is backed by atleast (2 service node|| 1 hardcoded) checkpoints // TODO(loki): Make this count the number of checkpoints backing this transfer
```

Comment:
This TODO would be nice, but I expect it is actually fairly tricky/expensive to calculate.

Reply:
Indeed. Non trivial atm, considering checkpoints can arrive out of order.